File: Backend/CareTag/src/main/java/com/example/CareTag/DTOs/VerifyResponse.java
package com.example.CareTag.DTOs;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@Data
@NoArgsConstructor
public class VerifyResponse {


      String token;
    String username;
    String RefreshToken;
}

File: Frontend/caretag/lib/Modules/auth/data/auth/verify_response.dart
class VerifyResponse {
  String? username;
  String? jwtToken;
  String? responseToken;

  VerifyResponse({this.username, this.jwtToken, this.responseToken});

  VerifyResponse.fromJson(Map<String, dynamic> json) {
    username = json['username'];
    jwtToken = json['token'];
    responseToken = json['RefreshToken'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['username'] = username;
    data['token'] = jwtToken;
    data['RefreshToken'] = responseToken;
    return data;
  }
}
